demos/virtual_ai_assistant_demo/default_personality.yaml

```diff
@@ -24,3 +24,26 @@ summarize_the_user_prompt: >
   Strictly do not mention any personal data like age, name, gender, contact, non-health information etc. when summarizing.
   Summarize the health-related concerns mentioned by the patient in this conversation or in the provided context.
   You must include information from the context if it's provided.
+
+title:
+  "# Adrishuo: A Virtual AI assistant running with OpenVINO
+
+  Instructions for use:  
+  
+
+  1. Attach the PDF or TXT file with an additional context (e.g prior examination report - optional; see \"Sample LLM Patient Records.pdf\" as an example) 
+
+
+  2. Record your question/comment using the first audio widget (\"Your voice input\") or type it in the textbox (\"Your text input\"), then click Submit  
+
+
+  3. Wait for the chatbot to response (\"Chatbot\")  
+
+
+  4. Discuss with the chatbot  
+
+
+  5. Click the \"Summarize\" button to make a summary  
+
+
+  **Note: This chatbot application is not intended to be used for medical purposes. It is for demonstration purposes only.**"
```

Comment:
```suggestion
title: |
  # Adrishuo: A Virtual AI assistant running with OpenVINO

  Instructions for use:  
  1. Attach the PDF or TXT file with an additional context (e.g prior examination report - optional; see \"Sample LLM Patient Records.pdf\" as an example) 
  2. Record your question/comment using the first audio widget (\"Your voice input\") or type it in the textbox (\"Your text input\"), then click Submit  
  3. Wait for the chatbot to response (\"Chatbot\")  
  4. Discuss with the chatbot  
  5. Click the \"Summarize\" button to make a summary  

  **Note: This chatbot application is not intended to be used for medical purposes. It is for demonstration purposes only.**
```

Reply:
Did you try this? If I don't keep spaces between lines Gradio APP will take that section in a paragraph instead of a list.

Reply:
Yes, the important thing is "|" in the first line. It changes the behaviour. Try it :P